### PR TITLE
Create amp-ad-custom.md.

### PR DESCRIPTION
**Pull request by glevitzky**
Add documentation for amp-ad-custom integration.

### CODE REVIEW

File: extensions/amp-ad-custom/amp-ad-custom.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+To make sure the template is valid AMPHTML, ad network must make an HTTP request to the cache URL, which returns a 4XX error, if invalid. This will also warm up the cache, and thereby speed up the first ad load. The cache URL can be converted from the canonical AMP URL following the rule described above (see detailed URL format <a href="https://developers.google.com/amp/cache/overview#amp-cache-url-format">here</a>).
+
+To update a template, an ad network just needs to update the page on their domain, then make another cache warm-up request.
+
+Note: at this stage, domains need to be whitelisted manually in AMP Cache to experiment with the feature.
+
+## Ad server
+
+The ad network needs to provide a new serving endpoint that returns an AMP CORS response in a JSON format:
```
Comment (lannka) [inline extensions/amp-ad-custom/amp-ad-custom.md]: AMP CORS is deprecated. Here we only need CORS

File: extensions/amp-ad-custom/amp-ad-custom.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+```js
+{
+  templateUrl: “https://adexample.com/amp_template_1.html”
```
Comment (lannka) [inline extensions/amp-ad-custom/amp-ad-custom.md]: please replace all the `“` & `”` to `"`

File: extensions/amp-ad-custom/amp-ad-custom.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    }
+  }
+}
+```
+
+The response requires a couple of custom headers:
+- custom headers for AMP CORS (obsolete, just do regular CORS)
+- AMP-Ad-Template-Extension: amp-mustache
+- Amp-Ad-Response-Type: template
+- Access-Control-Expose-Headers: AMP-Access-Control-Allow-Source-Origin
```
Comment (lannka) [inline extensions/amp-ad-custom/amp-ad-custom.md]: AMPCORS has been deprecated, this header is not needed.

File: extensions/amp-ad-custom/amp-ad-custom.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    type: “googleanalytics”,
+    config: {
+      ...
+    }
+  }
+}
+```
+
+The response requires a couple of custom headers:
+- custom headers for AMP CORS (obsolete, just do regular CORS)
```
Comment (lannka) [inline extensions/amp-ad-custom/amp-ad-custom.md]: only regular CORS needed now

**Comment by lannka** [comment]
@rsimha was it deleted accidentally?

**Comment by rsimha** [comment]
> @rsimha was it deleted accidentally?

Yes, accidental delete during main branch cleanup. Restored.

**Pull request closed without merging**